### ISSUE TITLE
Picking option in picker crashes application - iOS ONLY

### ISSUE THREAD

**Issue by LeoJHarris** [xamarin/Xamarin.Forms#11155]
iOS ONLY issue. Selecting option using picker causes app to crash. Seems somewhat specific to this scenario I have but the error log is below indicating the issue is the Xamarin.Forms.Platform.iOS.PickerRendererBase class. See attached example taken from the project I am having issues with, I tried to recreate in a sample project but was unable to so attached partial project itself.

```csharp
2020-06-22 16:53:06.340 MiniBarApp.iOS[258:13970] Unhandled managed exception: Object reference not set to an instance of an object (System.NullReferenceException)
  at Xamarin.Forms.Platform.iOS.PickerRendererBase`1[TControl].UpdateCharacterSpacing () [0x00006] in <b4a6bff6d9dc43cf83129db67d9306c1>:0 
  at Xamarin.Forms.Platform.iOS.PickerRendererBase`1+<>c__DisplayClass8_0[TControl].<OnElementChanged>b__0 (System.Object o, System.EventArgs a) [0x00072] in <b4a6bff6d9dc43cf83129db67d9306c1>:0 
  at UIKit.UIBarButtonItem+Callback.Call (Foundation.NSObject sender) [0x0000d] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.18.2.1/src/Xamarin.iOS/UIKit/UIBarButtonItem.cs:30 
  at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, System.IntPtr pr
incipal, System.IntPtr delegate) [0x00005] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.18.2.1/src/Xamarin.iOS/UIKit/UIApplication.cs:86 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.18.2.1/src/Xamarin.iOS/UIKit/UIApplication.cs:65 
  at MiniBarApp.iOS.Application.Main (System.String[] args) [0x00001] in C:\Users\Admin\source\repos\MiniBarApp\MiniBarApp\MiniBarApp.iOS\Main.cs:19 
```

### Steps to Reproduce

1. Click login button on login screen.
2. Select first option in the picker
3. App crashes.

### Expected Behavior

App to not crash

### Actual Behavior

App crashes.

### Basic Information

- Version with issue: 4.7.0.968
- Last known good version: 4.2.0.910310
- IDE: Visual Studio 2019 community
- Platform Target Frameworks: 
  - iOS:  13.5
- Affected Devices: iPad 3rd Generation

### Reproduction Link

[sample.zip](https://github.com/xamarin/Xamarin.Forms/files/4811239/sample.zip)

### Workaround

Downgrading to Xamarin.Forms 4.2.0.910310 seems fix the issue.

**Comment by raymonderasmus** [comment]
I have the exact same crash. Implementing PickerRenderer in Custom Renderer.

It seems like any Forms version > 4.2 crashes.

**Comment by dishantmshah** [comment]
@jsuarezruiz Is there any workaround for this until the PR is merged and available for V 5.0.0.  I have one important feature which is breaking due to this picker and I can't do any XF downgrade.
